src/node_watchdog.cc

```diff
@@ -422,12 +427,57 @@ SigintWatchdogHelper::~SigintWatchdogHelper() {
 SigintWatchdogHelper SigintWatchdogHelper::instance;
 
 namespace watchdog {
+
+static void CallWithTimeout(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  // WatchdogWrap.call(timeout, f)
```

Comment:
Unncessary comment?

Reply:
it just notes how the method is called. I think it's useful but I can remove it if needed.

Reply:
Perhaps maybe expand the comment so that's clear. Otherwise it just looks like old commented out code that was forgotten to be removed.